seq-library/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor/APTw_3T_001_2uT_36SincGauss_DC90_2s_braintumor.m

```diff
@@ -21,10 +21,10 @@
 
 %% sequence definitions
 % everything in defs gets written as definition in .seq-file
-defs.n_pulses      = 36              ; % number of pulses
-defs.tp            = 50e-3           ; % pulse duration [s]
-defs.td            = 5e-3            ; % interpulse delay [s]
-defs.Trec          = 3.5             ; % recovery time [s]
+defs.n_pulses      = 36              ; % number of pulses    %36
```

Comment:
Changes like this should not be comitted to keep the files clean and avoid unnecessary reviews

Reply:
Hi Ivan, too early for a merge request. We have to review this internally first.

@schuenke : I think dicom processing should be part of the eval scripts. in the end we provide with the seq files clear instructions how data can be generated.
now we want to compare an evaluation script  for real data and simulated data.

we can also agree to evaluate 4D python data, that is stored as different file format. but somehow we need the link of sim and real data in the same script.

we are happy to provide example data, we are also happy to generate 4D dicoms  or 4D stacks from simulation .